teacher.rb

```diff
@@ -1,15 +1,15 @@
-require "./person.rb"
+require './person'
 
 class Teacher < Person
-    def initialize(name = "Unknown", age, parent_permission == true, specialization)
-        super(name, age, parent_permission)
-        @specialization = specialization
-    end
+  def initialize(specialization)
+    super()
```

Comment:
- [ ] Kindly add all the parameters to initialize a `Teacher` object which should be age, name, specialization, and parent permission.
- [ ] Also when calling `super()`, you have to add all the parameters which are gotten from the parent class, that is:
```npm
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```